### PR TITLE
[Documentation] Revised getting started for better readability.

### LINKED ISSUE #2636: Clean up, rewrite/rephrase/re-edit documents for customized usages
For users who want to customize (e.g., use different versions of tensorflow) and build within the execution environment (e.g., run meson/ninja directly in a shell), the current documentation is messy, hard to follow, and not so ready for custom-install (e.g., run ninja install while there is another version of nnstreamer installed with apt).

We definitely need to clean them up.

### PR DESCRIPTION
**Pull request by myungjoo**
Do not mix up different build environments.
Seperate documents for better readability.

Added ```rm -rf build``` in spec file to make it consistent with the documentation.
Now, you may do ```gbs build --include-all``` even if you have built in your git repo.

Fixes #2636

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: Documentation/getting-started-ubuntu-debuild.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+## Getting Started: Ubuntu Debuilg/Pdebuild
```
Comment (wooksong) [inline Documentation/getting-started-ubuntu-debuild.md:1]:
```suggestion
## Getting Started: Ubuntu Debuild/Pdebuild
```

Typo here.

File: Documentation/getting-started-meson-build.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+## Getting Started: Meson/Ninja Build
+
+NNStreamer uses meson/ninja as its standard build envrionment; both Tizen(GBS-OBS/.rpm) and Ubuntu(pdebuild/.deb) builds uses meson/ninja.
+
+With this methos, you may build at the git repo directory and install nnstreamer binaries at your designated directories.
```
Comment (wooksong) [inline Documentation/getting-started-meson-build.md:5]:
```suggestion
With this method, you may build at the git repo directory and install nnstreamer binaries at your designated directories.
```

File: Documentation/getting-started-ubuntu-debuild.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+```
+
+To:
+```makefile
+ARCH := $(shell uname -m | sed -e 's/i[3-8]86/x86_32/')
+```
+
+
+
+***3. Prepare for debuild (installing required dependencies.***
```
Comment (wooksong) [inline Documentation/getting-started-ubuntu-debuild.md:86]:
```suggestion
***3. Prepare for debuild (installing required dependencies.)***
```

File: Documentation/getting-started-tizen.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+### Build with options
+
+***Enable full unit testing***
+```bash
+$ gbs build --define "unit_test 1"
+```
+
+***Get unit-test coverage report with the unit test for aarch64***
+```bash
+$ gbs build -A aarch64 --define "unit_test 1" --define "unittest-coverage"
```
Comment (wooksong) [inline Documentation/getting-started-tizen.md:27]:
```suggestion
$ gbs build -A aarch64 --define "unit_test 1" --define "testcoverage 1"
```

File: Documentation/getting-started-meson-build.md (hunk 2 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:5]: Applied. Thanks.

File: Documentation/getting-started-tizen.md (hunk 4 above)
Comment (myungjoo) [reply Documentation/getting-started-tizen.md:27]: Applied.

File: Documentation/getting-started-ubuntu-debuild.md (hunk 1 above)
Comment (myungjoo) [reply Documentation/getting-started-ubuntu-debuild.md:1]: Applied

File: Documentation/getting-started-ubuntu-debuild.md (hunk 3 above)
Comment (myungjoo) [reply Documentation/getting-started-ubuntu-debuild.md:86]: Applied

File: Documentation/getting-started-meson-build.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+Configure build in ```./build``` directory with default options.
+```bash
+$ meson build
+```
+
+For possible configurations, please refer to [meson_options.txt](../meson_options.txt).
+If you want to designate nnstreamer's binary locations, provide such with ```--prefix```, ```--sysconfdir```, ```--libdir```, ```--bindir```, and ```--include-dir```.
+For example, in order to install as general Linux system libraries you may configure:
+```bash
+$ mesno build ${whateveroptionyouwant....} --prefix=/ --sysconfdir=/etc --libdir=/usr/lib --bindir=/usr/bin --includedir=/usr/include
```
Comment (anyj0527) [inline Documentation/getting-started-meson-build.md:60]: minor typoe: `mesno -> meson`

File: Documentation/getting-started-meson-build.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+***3. Configure the build***
+
+Configure build in ```./build``` directory with default options.
+```bash
+$ meson build
+```
+
+For possible configurations, please refer to [meson_options.txt](../meson_options.txt).
+If you want to designate nnstreamer's binary locations, provide such with ```--prefix```, ```--sysconfdir```, ```--libdir```, ```--bindir```, and ```--include-dir```.
```
Comment (anyj0527) [inline Documentation/getting-started-meson-build.md:57]: minor typo: ```--include-dir``` -> ```includedir```

**Review by anyj0527 — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

File: Documentation/getting-started-meson-build.md (hunk 5 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:60]: Resolved

File: Documentation/getting-started-meson-build.md (hunk 6 above)
Comment (myungjoo) [reply Documentation/getting-started-meson-build.md:57]: Resolved
